ccm_web/package.json

```diff
@@ -14,103 +14,98 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./server/test/jest-e2e.json --detectOpenHandles"
+    "test:e2e": "jest --config ./server/test/jest-e2e.json"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
     "@kth/canvas-api": "^3.0.3",
-    "@material-ui/core": "^4.11.3",
+    "@material-ui/core": "^4.12.3",
     "@material-ui/icons": "^4.11.2",
-    "@nestjs/common": "^7.6.15",
-    "@nestjs/config": "^0.6.3",
-    "@nestjs/core": "^7.6.15",
-    "@nestjs/jwt": "^7.2.0",
-    "@nestjs/passport": "^7.1.5",
-    "@nestjs/platform-express": "^7.6.15",
-    "@nestjs/sequelize": "^0.2.0",
-    "@nestjs/swagger": "^4.8.0",
+    "@nestjs/axios": "^0.0.3",
+    "@nestjs/common": "^8.2.0",
+    "@nestjs/config": "^1.1.0",
+    "@nestjs/core": "^8.2.0",
+    "@nestjs/jwt": "^8.0.0",
+    "@nestjs/passport": "^8.0.1",
+    "@nestjs/platform-express": "^8.2.0",
+    "@nestjs/sequelize": "^8.0.0",
+    "@nestjs/swagger": "^5.1.4",
     "@react-hook/debounce": "^4.0.0",
-    "axios": "^0.21.1",
+    "axios": "^0.24.0",
     "class-transformer": "^0.4.0",
     "class-validator": "^0.13.1",
-    "connect-session-sequelize": "^7.1.1",
+    "connect-session-sequelize": "^7.1.2",
     "cookie-parser": "^1.4.5",
     "csurf": "^1.11.0",
     "express": "^4.17.1",
     "express-session": "^1.17.2",
     "got": "^11.8.2",
     "helmet": "^4.6.0",
-    "js-cookie": "^2.2.1",
+    "js-cookie": "^3.0.1",
     "keycode-js": "^3.1.0",
-    "ltijs": "~5.7.3",
-    "ltijs-sequelize": "~2.4.3",
+    "ltijs": "~5.8.2",
+    "ltijs-sequelize": "~2.4.4",
     "morgan": "^1.10.0",
-    "notistack": "^1.0.5",
-    "papaparse": "^5.3.0",
+    "mysql2": "^2.3.2",
```

Comment:
`mysql` was already available before because of `litjs`, but because we're using it without `ltijs`, it seems like we should specify it going forward.